InnerLoopAreas.props

```diff
@@ -3,6 +3,6 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!-- Feature to include for inner loop build -->
   <PropertyGroup Condition="$(SolutionName) == 'MUXControlsInnerLoop'">
-    <FeatureTabViewEnabled>true</FeatureTabViewEnabled>
+    <FeatureProgressRingEnabled>true</FeatureProgressRingEnabled>
```

Comment:
Should this file be git ignored ?

Reply:
@teaP I remember you investigated into this?